src/Editor/GUI/Editors/AnimationClipEditor.hpp

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+#include "../FileSelector.hpp"
+#include "Util/AssetConverter.hpp"
+#include "../../Resources.hpp"
+
+namespace Animation {
+    class AnimationClip;
+}
+
+namespace GUI {
+    class AnimationClipEditor {
+        public:
+            /// Constructor.
+            AnimationClipEditor();
+
+            /// Show the editor.
+            void Show();
+
+            /// Returns the active animation clip.
+            /**
+             * @return The active animation clip. 
+             */
+            Animation::AnimationClip* GetAnimationClip();
+
+            /// Set the skeleton to edit
+            /**
+            * @param folder Resource folder containing the animation clip.
```

Comment:
As mentioned in the cpp file, this parameter is never used.
Asterisks are not aligned.